tests/test_gitbook.py

```diff
@@ -17,55 +18,56 @@ def _path(d):
 
```

Comment:
Now that we have a class, these lines should probably go in `setUpClass` rather than in the global space.